server/src/main/java/com/skillmagnet/User/User.java

```diff
@@ -31,6 +36,10 @@ public class User {
 
     private LocalDateTime lastLogin;
 
+    @OneToMany
+    @JoinColumn(name = "lesson_id")
+    private Set<Lesson> lessonsCompleted;
```

Comment:
new set that is hooked to lesson table

Reply:
I questioned this at first but I'm content with it if makes the other calculations easier.